### ISSUE TITLE
Create REST blueprint not returning full expected response.

### ISSUE THREAD

**Issue by gotmikhail** [balderdashy/sails#2427]
I noticed I was not getting the same response on a post as I was on a get. Specifically, associations.

Referencing 'Create new record while associating w/ another new record'
https://github.com/balderdashy/sails-docs/blob/master/reference/blueprint-api/Create.md

I recreate the scenario from the documentation with `sails new testProj`, then `sails generate api pony` and `sails generate api pet`, then update the models to have a one-to-one association.

Pony
```javascript
module.exports = {
    attributes: {
        pet: {
            model: 'pet'
        }
    }
};
```
Pet
```javascript
module.exports = {
    attributes: {
        pony: {
            model: 'pony'
        }
    }
};
```
Then, `sails lift` away.

`POST /pony`
JSON Req
```javascript
{
  "name": "Pinkie Pie",
  "hobby": "ice skating",
  "pet": {
    "name": "Gummy",
    "species": "crocodile"
  }
}
```
JSON Res
```javascript
{
  "name": "Pinkie Pie",
  "hobby": "ice skating",
  "pet": 1,
  "createdAt": "2014-11-21T19:18:09.161Z",
  "updatedAt": "2014-11-21T19:18:09.161Z",
  "id": 1
}
```
As you can see, the `res`ponse did not include the full pet object, just its id. In the documentation, it shows the full object being returned.

The entries and the association were definitely created successfully, 
`GET - /pony/1`
```javascript
{
  "pet": {
    "name": "Gummy",
    "species": "crocodile",
    "createdAt": "2014-11-21T19:18:09.157Z",
    "updatedAt": "2014-11-21T19:18:09.157Z",
    "id": 1
  },
  "name": "Pinkie Pie",
  "hobby": "ice skating",
  "createdAt": "2014-11-21T19:18:09.161Z",
  "updatedAt": "2014-11-21T19:18:09.161Z",
  "id": 1
}
```

Is this an error in documentation, or an error in code? Either way, is there a way we can specify that we want the associations to be returned?

One-to-one associations are just showing up as the linked id, many to many do not show up at all. I was looking into the `toObject()` function for `waterline` for another issue when I saw the `showJoins` option; it looks like it might be related to that.

**Comment by Dallas62** [comment]
I noticed that too

https://github.com/balderdashy/sails-docs/issues/384

**Comment by CWyrtzen** [comment]
Is this still an issue?

**Comment by sailsbot** [comment]
Thanks for posting, @gotmikhail.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
